core/platform/models.py

```diff
@@ -468,6 +503,7 @@ def import_cache_services(cls):
 
     @classmethod
     def import_taskqueue_services(cls):
+        # type: () -> ModuleType
```

Comment:
Is it possible to specify which module type?

Reply:
No actually, I tried it. But it was not possible to do it.